### PR TITLE
Fix images quality

### PR DESCRIPTION
**Pull request by VladislavSokov**

### CODE REVIEW

File: src/showcases.tsx [hunk 1]
```diff
@@ -189,7 +189,7 @@ const showcases: Showcase[] = [
           Prevent account sharing to <span className="oval">stop</span> financial <span className="stripe">losses</span>
         </h1>
       ),
-      bannerTopImageSrc: "/img/showcases/case/showcase-2-cybersicurity.webp",
+      bannerTopImageSrc: "/img/showcases/case/showcase-2-cybersicurity.jpg",
```
Comment (ka8725) [inline src/showcases.tsx:192]: cybers**e**curity

File: src/app/showcases/[slug]/page.tsx [hunk 2]
```diff
@@ -67,7 +67,14 @@ export default function ShowcasePage() {
             {showcase.body.bannerTopTitle}
           </div>
           <div className="img-wrapper">
-            <Image src={`${showcase.body.bannerTopImageSrc}`} alt="Banner Image" width={687} height={377} />
+            <Image
+              src={`${showcase.body.bannerTopImageSrc}`}
+              priority
```
Comment (ka8725) [inline src/app/showcases/[slug]/page.tsx:72]: @VladislavSokov which formats will be supported with this config?

**Comment by ka8725** [comment]
@VladislavSokov will it work in production? No additional changes needed?

**Comment by VladislavSokov** [comment]
> @VladislavSokov will it work in production? No additional changes needed?

This should work.

**Comment by ka8725** [comment]
@VladislavSokov 

My browser supports webp, but it shows only jpg:

image

**Comment by ka8725** [comment]
Test

**Pull request merged**